src/WCSimWCTrigger.cc

```diff
@@ -32,7 +32,7 @@
 #endif
 
 const double WCSimWCTriggerBase::offset = 950.0; // ns. apply offset to the digit time
-const double WCSimWCTriggerBase::LongTime = 1E6; // ns = 1ms. event time
```

Comment:
Is this reasonable to change the LongTime variable to 10ms? This is the value that will be in the actual trigger?

Reply:
`LongTime` is used in two places
* `GetPostTriggerWindow()` (and, as of this PR, `GetPreTriggerWindow()` only for the NoTrigger trigger.
* Setting the end (and, as of this PR, start) times of the loop in `AlgNDigits()`, before the start/end time have been found by looping over all digits.

Therefore the effect of `LongTime` doesn't change the trigger, other than making it more robust in instances of
* long time windows
* not setting the time window simulated to start at 0 (particle gun & muline defaults)

I don't remember exactly why I choose 10 times more (but recall the neutron capture lifetime on H is about 0.2 ms). An even greater range would be better.